.github/workflows/build_releasecandidate.yml

```diff
@@ -0,0 +1,91 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI – Release Candidate
+
+# Controls when the action will run. Triggers the workflow on push events if the pushed
+# tag is a release candidate
+on:
+  push:
+    tags:
+      - "[2-9][0-9][0-9][0-9].[0-9]+-RC[0-9]+"
```

Comment:
You are really, really thinking far ahead of time with this regex, right? Maybe "20[1-9][0-9]" would have sufficed :D